pages/deploy/deploy_aws.md

```diff
@@ -518,6 +518,25 @@ diagram.
 With this pipeline we can now deploy the infrastructure to the deployment account(s). Navigate to AWS CodePipeline
 in the tooling account and check the status of your pipeline.
 
+## 12. Setting SES for Email Notifications
+
+Please follow instructions from below if you have enabled email notification by switching the email.active config to true in the `share_notifications` feature under `datasets` module. 
```

Comment:
Can we add here that we are referring to the `config.json`? Deployment is quite confusing for new data.all-ers so I would make it as clear as possible

Reply:
Yes. I made changes to specify where exactly the configuration is present.